### PR TITLE
style(ndm): Fix lint errors in scsicommands.go

### LINKED ISSUE #105: Fix lint errors in scsicommands.go
File: [scsicommands.go](https://github.com/openebs/node-disk-manager/blob/master/pkg/smart/scsicommands.go)

```
Line 31: warning: comment on exported type CDB6 should be of the form "CDB6 ..." (with optional leading article) (golint)
Line 34: warning: exported type CDB10 should have comment or be unexported (golint)
Line 35: warning: exported type CDB16 should have comment or be unexported (golint)
```

### PR DESCRIPTION
**Pull request by jeanmorais**
Fixes #105

Signed-off-by: Jean Morais jeancdemorais@gmail.com

### CODE REVIEW

**Comment by ashishranjan738** [comment]
@jeanmorais can u please sign ur PR as tests are failing.

File: pkg/smart/scsicommands.go [hunk 1]
```diff
@@ -30,8 +30,14 @@ const (
 
 // SCSI Command Descriptor Block types are the various type of scsi cdbs which are used
 // to specify the various parameters or data required to send a particular scsi command
+
+// CDB6 type
```
Comment (ashishranjan738) [inline pkg/smart/scsicommands.go]: Can this be elaborated?

Comment (jeanmorais) [reply pkg/smart/scsicommands.go]: Yes. How elaborate? Something like this is enough?

"// CDB6 type represents 6 byte CDB"
...

Comment (ashishranjan738) [reply pkg/smart/scsicommands.go]: or "// CDB6 is a slice of 6 BYTE"
... ?

Comment (jeanmorais) [reply pkg/smart/scsicommands.go]: > or "// CDB6 is a slice of 6 BYTE"
> ... ?

Ok. I'll do this.
Thanks!

**Comment by ashishranjan738** [comment]
@jeanmorais you need to sign your commit. For doing that please commit your changes -s flag.

**Comment by sagarkrsd** [comment]
@jeanmorais Its because your first commit is not signed !

**Comment by ashishranjan738** [comment]
Try doing a force push or close it and raise one more PR.

File: pkg/smart/scsicommands.go [hunk 2]
```diff
@@ -30,8 +30,14 @@ const (
...
 // SCSI Command Descriptor Block types are the various type of scsi cdbs which are used
 // to specify the various parameters or data required to send a particular scsi command
+
+// CDB6 is a slice of 6 byte
 type CDB6 [6]byte
+
+// CDB10 is a slice of 10 byte
 type CDB10 [10]byte
+
+//CDB16 is a slice of 16 byte
```
[the remark this answers is not in the document]
Comment (vharsh) [reply pkg/smart/scsicommands.go]: @jeanmorais Please replace `slice` with `array` for all of the above.

**Comment by jeanmorais** [comment]
@sagarkrsd The last commits is signed. Why is the signature failing?

**Comment by sagarkrsd** [comment]
@jeanmorais Right, your last commit is signed but your first commit is not signed .
So you need to rebase all the commits into one commit and then you can sign it using `git commit --amend` .

**Pull request merged**